clients/fides-js/src/lib/cookie.ts

```diff
@@ -331,13 +331,18 @@ export const makeConsentDefaultsLegacy = (
 
 /**
  * Given a list of cookies, deletes them from the browser
+ * Optionally removes subdomain cookies as well
  */
-export const removeCookiesFromBrowser = (cookiesToRemove: CookiesType[]) => {
+export const removeCookiesFromBrowser = (
+  cookiesToRemove: CookiesType[],
+  removeSubdomainCookies: boolean = false,
+) => {
   cookiesToRemove.forEach((cookie) => {
-    cookies.remove(cookie.name, {
-      path: cookie.path ?? "/",
-      domain: cookie.domain,
-    });
+    const { hostname } = window.location;
+    cookies.remove(cookie.name);
```

Comment:
I'm thinking for this line we could keep the previous logic of: 
```
cookies.remove(cookie.name, {
      path: cookie.path ?? "/",
      domain: cookie.domain,
    });
```

This way, if we ever were somehow deleting compass cookies before (unlikely but still), we won't change functionality unintentionally. what do you think? Then when we follow-up with handling compass cookies, too, we can update this bit.